implementations/rust/Cargo.toml

```diff
@@ -3,5 +3,8 @@
 members = [
   "crates/ockam-vault-sys",
   "crates/ockam",
-  "vault"
+]
+
+exclude = [
```

Comment:
Why is vault excluded?

Reply:
For now, I'm building it separately to avoid some library conflict issues.